website/content/docs/connect/config-entries/http-route.mdx

```diff
@@ -446,6 +513,151 @@ The following table describes the parameters for `path`:
+  - `reset`
+  - `connect-failure`
+  - `envoy-ratelimited`
+  - `retriable-4xx`
+  - `refused-stream`
+  - `cancelled`
+  - `deadline-exceeded`
+  - `internal`
+  - `resource-exhausted`
+  - `unavailable`
+
+### `Rules[].Filters{}.RetryFilter{}.RetryOnStatusCodes`
+
+Specifies a list of integers for [HTTP response status codes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status) that trigger a retry request.
+
+#### Values
+
+- Default: None
+- Data type: List of integers
+
+### `Rules[].Filters{}.TimeoutFilter`
+
+Specifies timeout settings for routes from an API gateway listener to the destination service in Consul service mesh.
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+The following table describes the settings you can configure in the `TimeoutFilter` map:
+
+| Parameter | Description | Data type | Default |
+| ---       | ---         | ---       | ---     |
+| `IdleTimeout` | Specifies the total amount of time permitted for the request stream to be idle. | Integer | `0` |
+| `RequestTimeout` | Specifies the total amount of time in nanoseconds, including retry attempts, Consul permits for the entire downstream request to be processed. | Integer | `0` |
+
+### `Rules[].Filters{}.JWT`
+
+Specifies a block of JWT verification configurations to apply to the route. These route-specific settings have precedence over default configurations defined for listeners the route attaches to on the API gateway.  Refer to [`Listeners[].default{}.JWT`](/consul/docs/connect/config-entries/api-gateway#listeners-default-jwt) for additional information.
+
+#### Values
+
+- Default: None
+- Data type: Map
+
+### `Rules[].Filters{}.JWT{}.Providers`
+
+Specifies a list of JWT provider configurations to apply to the route. A provider configuration contains the name of the provider and claims.
+
+#### Values
+
+- Default: None
+- Data type: List of maps
+
+The following table describes the parameters you can specify in a member of the `Providers` list:
+
+| Parameter | Description | Data type | Default |
+| ---       | ---         | ---       | ---     |
+| `Name` | Specifies the name of the provider. | String | None |
+| `VerifyClaims` | Specifies a list of paths and a value that define the claim. Consul verifies requests that match the claims declared in the listener JWT configuration and allow the request through the gateway. The `VerifyClaims` map specifies the following settings: <ul><li>`Path`: Specifies a list of one or more registered or custom claims.</li><li>`Value`: Specifies the expected value of the claim.</li></ul> | Map | None |
+
+Refer to [Configure JWT verification settings](#configure-jwt-verification-settings) for an example configuration.
+
+### `Rules[].ResponseFilters`
+
+Specifies the list of HTTP-based filters used to modify a response prior to routing it to the calling client.
+
+#### Values
+
+- Default: none
+- Data type: Map that contains the following fields:
+  - `Headers`
+
+### `Rules[].ResponseFilters.Headers[]`
```

Comment:
@nathancoleman here too

Reply:
We should use the appropriate bracket type in the heading (either " . . . ResponseFilters{}. . . " or " . . . ResponseFilters[] . . ."

Reply:
Gotcha, there was inconsistencies earlier in the document. It is a map, so I should be using `ResponseFilters{}`. I'll change it in the other places I saw it too.

Reply:
I'm wrong. It's a list.